### DISCUSSION TITLE
Spacer Card

### DISCUSSION THREAD

**Discussion by GizmoGoody** [piitaya/lovelace-mushroom#783] [Ideas]
While creating a remote control using a combination of stack-in-card, vertical-stack, horizontal-stack, and grid cards, I just used an entity card with a null entity/icon to build spacing (e.g. above and below the left and right arrows), but I kept wondering why we don't have a Mushroom Spacer Card.

Thoughts? Does anyone add space in a different way already?

![image](https://user-images.githubusercontent.com/46235745/197409545-487c5b2b-fe1d-4054-86cb-38f35ed3e0db.png)

**Comment by piitaya** [comment]
Hello 🙂 There is no spacer card because Mushroom was not designed to be used with stack-in-card or card-mod. The focus in on official cards support (grid, vertical/horizontal stack).
Your example is a very advanced use case. Why not just use something like this : https://github.com/marrobHD/tv-card ?

**Reply by GizmoGoody** [reply]
The use case still stands with non-custom cards like grid and horizontal/vertical stack. 😀 

I don't have anything nice to say about the look and static layout of that TV card you posted.

I'm creating my own for the same reasons I also use Mushroom Cards 😉 - customization, functionality, and pleasing aesthetics.

A spacer card seemed like an easy thing to add, so I figured I'd propose the idea.

**Reply by piitaya** [reply]
May be https://github.com/thomasloven/lovelace-layout-card#gap-card ?

**Reply by GizmoGoody** [reply]
Here's another use case, but with Mushroom Chips

![image](https://user-images.githubusercontent.com/46235745/202727201-3aebabc6-c96a-40b9-9017-c938878f942f.png)

I have a view with all of my rooms laid out in a grid. In each room, the top section is reserved for Echo devices if they're present. But, if they're not present, I'd like to keep a blank row the same size as Chip Cards to maintain a consistent look (height of the room's card). Currently, I can put a placeholder entity card as pictured, but I'd prefer that space to be completely transparent. If there was a Spacer Chip, I could use that instead. As it stands, I think the next best way is to figure out the card-mod syntax to do the same thing.

**Reply by antonio1475** [reply]
Mind sharing the yaml for those room cards?
Thank you

**Comment by antonio1475** [comment]
https://github.com/custom-cards/button-card

```
- type: custom:button-card
  color_type: blank-card
  styles:
    card:
      - height: 15px
```
?
